script.js

```diff
@@ -2,6 +2,8 @@
 //TODO: account for website discounts
 //TODO: use user location to calculate local tax and display local currency
 
+const allTaxRates = require('./sales_tax_rates.json');
```

Comment:
Do you need to include the minified package from [require.js](https://requirejs.org/docs/api.html#jsfiles) to allow require? Or else we will have to switch to a node environment and can't just serve a static html site

Reply:
I'm getting errors on my end from the console, stopping loading.
<img width="460" alt="Screen Shot 2021-08-05 at 11 36 44 PM" src="https://user-images.githubusercontent.com/65370631/128466997-b7a914bd-9388-4e2e-82bf-48175e09048d.png">